### DISCUSSION TITLE
Add custom resources to export

### DISCUSSION THREAD

**Discussion by petrroda** [apache/camel-karavan#721] [Q&A] [answered]
Hi,

how can I add custom resources like Velocity templates to the exported app?

Greets
Petr

**Comment by mgubaidullin** [comment]
I believe it is `camel.jbang.classpathFiles` property in `application.properties`

https://camel.apache.org/manual/camel-jbang.html#_configuring_exporting

**Reply by petrroda** [reply]
@mgubaidullin 

Hi, 

when generating a Quarkus export then the specified resources are copied. 
But when running with JBang run, then I cannot access them.

Export as Quarkus:

![image](https://user-images.githubusercontent.com/12616511/232207408-35980d7a-38c7-4fb4-a7bb-7d038cb04da6.png)

YAML-Route:
                steps:
                  - to:
                      uri: velocity://insert_sat.vm
                      id: to-860f
                      description: Sat-Template

Stacktrace when testing with JBang:

java.io.FileNotFoundException: Cannot find resource: insert_sat.vm for URI: insert_sat.vm
        at org.apache.camel.support.ResourceHelper.resolveMandatoryResourceAsInputStream(ResourceHelper.java:115) ~[camel-support-3.20.3.jar:3.20.3]
        at org.apache.camel.component.ResourceEndpoint.loadResource(ResourceEndpoint.java:111) ~[camel-support-3.20.3.jar:3.20.3]

**Reply by davsclaus** [reply]
Do you run jbang _after_ you export to quarkus or before?

**Reply by petrroda** [reply]
After, is that relevant?

**Reply by davsclaus** [reply, marked as answer]
Yes because when you export, you _leave_ jbang and you use quarkus standard, if you want to run, then use `mvn quarkus:dev`.

jbang is not for running standard quarkus projects where the source code is structured in maven (or gradle).
